web/assets/js/actions.ts

```diff
@@ -247,6 +247,17 @@ class Actions {
     });
   }
 
+  /**
+   * Checks if the given game timed out
+   * @param gameID the ID of the running game
+   */
+  public async getTimeoutState(gameID: string) {
+    // Fetch the game
+    const game: Game = await this.getGame(gameID);
+
+    return game.state === GameState.TIMEOUT;
+  }
```

Comment:
change this to isGameOngoing or isGameOpen, and check instead for Game.state == GameState.OPEN

we cannot make a claim a timeout on any finished game

Reply:
@alexiscolin

Reply:
Renamed:

[a9aedee](https://github.com/gnolang/gnochess/pull/54/commits/a9aedee7c52e359708c303d217a81eb67f80056f)